ReleaseNotes.md

```diff
@@ -1,3 +1,15 @@
+### New in 0.21.1 (released 2016/07/29)
+
+**Features**
+
+Due to an programming error in the tool to generate these release notes, additional
+features were not properly documented for the previous release:
+
+ - Reactions preview API support for issues, commit comments and PR comments - #1335, #1405 via @maddin2016 
```

Comment:
There are actually 4 (not 3) items that Reactions are supported for:  commit comments, issues, issue comments, pull request review comments